### PR TITLE
Image compression: initial version of the entropy coder.

### PR DESCRIPTION
**Pull request by damienv-gh**

### CODE REVIEW

File: compression/entropy_coder/core/code_loader.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  Args:
+    input_config: An InputConfig proto containing the input configuration.
+    batch_size: Output batch size of examples.
+
+  Returns:
+    A batched tensor of binary codes.
+  """
+  data = input_config.data
+
+  # TODO(damienv): Possibly use multiple files (instead of just one).
```
Comment (nickj-google) [inline compression/entropy_coder/core/code_loader.py]: Do you want to remove this TODO, or place your github name here?

File: compression/entropy_coder/lib/blocks_std.py [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+    with self._BlockScope():
+      self._linear_block_factory = linear_block_factory
+      self._depth = depth
+      self._initializer = initializer
+      self._matrices = None
+
+      self._bias = BiasAdd(bias) if bias else PassThrough()
+      self._act = act if act else PassThrough()
+
+  # TODO(sjhwang): Stop using **kwargs, if we ever switch to python3.
```
Comment (nickj-google) [inline compression/entropy_coder/lib/blocks_std.py]: Remove TODO or add your github username here?

File: compression/entropy_coder/core/code_loader.py (hunk 1 above)
Comment (damienv-gh) [inline compression/entropy_coder/core/code_loader.py]: Done

File: compression/entropy_coder/lib/blocks_std.py (hunk 2 above)
Comment (damienv-gh) [inline compression/entropy_coder/lib/blocks_std.py]: Done

**Pull request merged**